superduperdb/components/datatype.py

```diff
@@ -89,27 +96,13 @@ class DataType(Component):
     artifact: bool = False
     reference: bool = False
     directory: t.Optional[str] = None
+    encodable_cls: t.Optional[t.Type['_BaseEncodable']] = None
```

Comment:
This will create problems with saving. Should replace with a reference as `str`.

Reply:
Done